static/app/components/issues/compactIssue.tsx

```diff
@@ -166,28 +170,20 @@ const CompactIssue = createReactClass<Props, State>({
     if (issue.status === 'ignored') {
       className += ' isIgnored';
     }
-    if (this.props.statsPeriod) {
-      className += ' with-graph';
-    }
 
     return (
-      <IssueRow className={className} onClick={this.toggleSelect}>
+      <IssueRow className={className}>
         <CompactIssueHeader
           data={issue}
           organization={organization}
           projectId={issue.project.slug}
           eventId={this.props.eventId}
         />
-        {this.props.statsPeriod && (
```

Comment:
Just to be clear, you intend to remove `statsPeriod` here and removing it doesn't change the product (or it does but it's fine)?

Reply:
I meant to remove this as it was never enabled. The only place it was used was in monitors and the value was '0'.